### Association between Device Type and Customer Satisfaction

#### 1. State the Hypotheses:

###### H0:There is no significant association between the type of device purchased and the customer satisfaction level.
###### H1:There is a significant association between the type of device purchased and the customer satisfaction level.

In [4]:

import numpy as np
from scipy.stats import chi2_contingency, chi2

# Observed data
# Rows → Satisfaction Levels
# Columns → Device Types (Smart Thermostat, Smart Light)
data = np.array([
    [50, 70],   # Very Satisfied
    [80, 100],  # Satisfied
    [60, 90],   # Neutral
    [30, 50],   # Unsatisfied
    [20, 50]    # Very Unsatisfied
])

# Perform Chi-Square Test
chi2_stat, p_val, dof, expected = chi2_contingency(data)

# Set significance level
alpha = 0.05
critical_value = chi2.ppf(1 - alpha, dof)

print("Chi-Square Test for Association\n")
print(f"Observed Data:\n{data}\n")
print(f"Expected Frequencies (if H₀ true):\n{expected.round(2)}\n")
print(f"Chi-Square Statistic = {chi2_stat:.3f}")
print(f"Degrees of Freedom   = {dof}")
print(f"Critical Value (α=0.05) = {critical_value:.3f}")
print(f"p-value = {p_val:.3f}\n")

if chi2_stat > critical_value:
    print(" Reject Null Hypothesis (H₀)")
    print("→ Significant association between Device Type and Satisfaction.\n")
else:
    print("Fail to Reject Null Hypothesis (H₀)")
    print("No significant association between Device Type and Satisfaction.\n")

Chi-Square Test for Association

Observed Data:
[[ 50  70]
 [ 80 100]
 [ 60  90]
 [ 30  50]
 [ 20  50]]

Expected Frequencies (if H₀ true):
[[ 48.  72.]
 [ 72. 108.]
 [ 60.  90.]
 [ 32.  48.]
 [ 28.  42.]]

Chi-Square Statistic = 5.638
Degrees of Freedom   = 4
Critical Value (α=0.05) = 9.488
p-value = 0.228

Fail to Reject Null Hypothesis (H₀)
No significant association between Device Type and Satisfaction.



#### Conclusion

##### The calculated Chi-Square statistic (χ²) is 5.64, while the critical value at a 5% significance level with 4 degrees of freedom is 9.488. Since the calculated value (5.64) is less than the critical value (9.488), and the p-value (0.227) is greater than the significance level (0.05), we fail to reject the null hypothesis (H₀). This indicates that there is no significant association between the type of device purchased (Smart Thermostat or Smart Light) and the customer satisfaction level. The observed differences in satisfaction counts between the two device types are small enough to be explained by random variation rather than an actual relationship. Hence, we conclude that the type of smart home device does not significantly influence customer satisfaction.